acestep/training_v2/ui/progress.py

```
@@ -13,15 +14,48 @@
+        super().__init__(level=logging.INFO)
+        self._messages = messages
+        self._live = live
+
+    def emit(self, record: logging.LogRecord) -> None:
+        try:
+            msg = record.getMessage()
+            self._messages.append(msg)
+            if len(self._messages) > 20:
+                self._messages.pop(0)
+        except Exception:
+            pass
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)